data/FloraOfAustralia_2022/metadata.yml

```diff
@@ -0,0 +1,219 @@
+source:
+  primary:
+    key: FloraOfAustralia_2022
+    bibtype: Online
+    year: 2022
+    author: Australian Biological Resources Study
+    title: Flora of Australia, Australian Biological Resources Study, Canberra.
+    url: http://www.ausflora.org.au
+contributors:
+  data_collectors:
+  - last_name: Coleman
```

Comment:
We'd better follow up with each flora/group and find out who is best to credit here. 

David can come in as last collector. I think we have the option to add notes for authors. In that case, good to specific what David's role was ""E.g. Responsible for extracting data from online resource"

Reply:
I've added notes below David's name and more importantly removed these datasets from this branch while we contact the floras